static/src/javascripts/es6/projects/common/modules/crosswords/anagram-helper/clue-input.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default class ClueInput extends React.Component {
+    componentDidMount () {
+        React.findDOMNode(this).focus();
+    }
+
+    componentWillReceiveProps (props) {
+        // focus on reset
+        if (props.value === '') {
+            const node = React.findDOMNode(this);
+            setTimeout(node.focus.bind(node), 4);
```

Comment:
yep, it not being mounted was the reason, but I can actually use componentDidUpdate (called after mounting) instead and check the current props rather than the one being passed in. thanks!